file/reader.go

```diff
@@ -84,6 +84,21 @@ func yamlToJSON(m map[interface{}]interface{}) map[string]interface{} {
 		switch v2 := v.(type) {
 		case map[interface{}]interface{}:
 			res[fmt.Sprint(k)] = yamlToJSON(v2)
+		case []interface{}:
```

Comment:
Why was this necessary?

(I've a vague memory that this is what ghodss/yaml took care of for us).